app/templates/suppliers/companies_house_number.html

```diff
@@ -28,12 +28,13 @@
 
   <div class="grid-row">
     <div class="column-two-thirds">
-      <div class="hint">
-        <p>This is a unique 8-character number given to your organisation by Companies House.</p>
-        <p><a href="https://beta.companieshouse.gov.uk/">Visit Companies House to get your number</a></p>
-      </div>
       <form method="POST" action="{{ url_for('.companies_house_number') }}">
         <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
+        {% set question_advice =
+            'This is a unique 8-character number given to your organisation by Companies House.
+            \n\n<a href="https://beta.companieshouse.gov.uk/">Visit Companies House to get your number</a>
```

Comment:
If you write pure HTML into this field, it gets interpreted really strangely by markdown when you start  using linebreaks.  This way, it's easier for humans to maintain, though I agree that it's not ideal.